server/api/router.go

```diff
@@ -194,6 +204,11 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) (*mux.
 	apiRouter.Handle("/debug/pprof/block", pprof.Handler("block"))
 	apiRouter.Handle("/debug/pprof/goroutine", pprof.Handler("goroutine"))
 
+	// swagger docs
+	swaggerHandler := swaggerserver.Handler()
+	apiRouter.PathPrefix("/swagger/").Handler(swaggerHandler)
```

Comment:
Is it better to add a `HandlerBuilder` to `main.go` and access it directly via `/swagger`?

Reply:
`/pd/swagger` or `/swagger`?